.github/workflows/build_e2e_test_artifacts.yml

```diff
@@ -96,6 +96,10 @@ jobs:
         run: gcloud storage cp "${TF_BINARIES_GCS_ARTIFACT}" "${TF_BINARIES_ARCHIVE}"
       - name: "Extracting TF binaries archive"
         run: tar -xf "${TF_BINARIES_ARCHIVE}"
+      - name: "Enlarge disk"
+        run: |
+          sudo mount -o remount,size=200G /runner-root
```

Comment:
Ah, so this is tricky. That disk is a RAM disk. The whole machine has 360GB of RAM, so this is using more than half of it. Are you sure that's what we want?

Generally I think we probably don't want workflows messing with the disk. I've held off on switching this to a local SSD like we have for the GPU machines because the n1-standard-96 requires having something like 24 local SSD if you have any, which is a bit much. We could also just try switching this to having persistent SSD on the side.

Reply:
Thanks for the update

Reply:
https://github.com/openxla/iree/pull/13079 has been rolled out and all the runners should have 1TB of disk now